clients/deltalake/pom.xml

```diff
@@ -308,6 +308,9 @@
                 <goals>
                   <goal>doc-jar</goal>
                 </goals>
+                <configuration>
+                  <scalaVersion>${scala2.12.version}</scalaVersion>
```

Comment:
does it not warrant a comment why only for this usage of the plugin we have to use a specific version and in the others we dont?

Reply:
all of the other scala modules were already explicitly using scala 2.12, this is just aligning the deltalake module